web-js/upload-shapefile-to-agol/index.html

```diff
@@ -0,0 +1,306 @@
+				function (response) {
+					addFeatureLayer(url + "?token=" + token);
+					$("#pendingText").css("visibility", "hidden");
+				},
+				function (error) {
+					setTimeout(function() {
+						checkServiceStatus(url);
+					}, 1500);
+				}
+			);
+		}
+		
+		function addFeatureLayer(url) {
+			if(featureLayer) {
+				map.removeLayer(featureLayer);
+				featureLayer = null;
+			}
+			featureLayer = new FeatureLayer(url);
+			map.addLayers([featureLayer]);
+		}
+		
+    });
+    </script>
+</head>
+
+<body class="tundra">
+	<div class="content-container">
+		<div class="nav-container">
+			<nav class = "navbar" role = "navigation">
+				<div class = "navbar-header">
+					<p class = "navbar-brand">Upload Shapefile to ArcGIS Online</p>
+				</div>
+				<div>
+					<p id="signedInUser" class = "navbar-text navbar-right">
+						Signed in as: 
+						<a id="currentUser" class = "navbar-link" style="color: #fff !important" ></a>
+					</p>
+				</div>
+			</nav>
+		</div>
+		<div class = "flex-box">
+			<div class = "form-container column">
+				<div class = "form-group">
+					<label for = "org" class="label-config">Organization</label>
+					<input type = "text" class = "form-control input-elements" id = "org" placeholder = "Ex: ess.maps.arcgis.com" >
+				</div><br/>
+				<div class = "form-group">
+					<label for = "username">Username</label>
+					<input type = "text" class = "form-control input-elements" id = "username" placeholder = "Enter Username" >
+				</div><br/>
+				<div class = "form-group">
+					<label for = "password">Password</label>
+					<input type = "password" class = "form-control input-elements" id = "password" placeholder = "Enter Password" >
+				</div><br/>
+				<div class = "form-group">
+					<button id = "signInBtn" type = "submit" class = "btn btn-primary">Sign In</button><br/>
+				</div><br/>
+				<div class = "form-group">
+					<label for = "title">Item Title</label>
+					<input type = "text" class = "form-control input-elements" id = "title" placeholder = "Enter Item Name">
+				</div><br/>
+				<form id="data" enctype="multipart/form-data" method="post" role = "form">
+					<div class = "form-group" style="margin-bottom: 27px !important">
+						<label for = "file">Select File</label>
+						<input type = "file" id = "file" name = "file" class="btn input-elements" style="margin-right: 15px">
+					</div><br/>
+					<div class = "checkbox" style="margin-top: 0px">
+						<label><input id="featureServiceCB" type = "checkbox" checked>Create FeatureService</label>
+					</div><br/>
+					<div class = "checkbox" style="margin-bottom: 27px; margin-top: 0px">
+						<label><input id="addServiceCB" type = "checkbox" checked>Add Service to Map</label>
+					</div><br/>
+					<button type = "submit" class = "btn btn-primary" >Submit</button>
+				</form>
+			</div>
+			<div id="map" class="column">
+				<h1 id="pendingText" class="pending-text">Pending</h1>
+			</div>
+		</div>
+	</div>
+</body>
+</html>
```

Comment:
this sample is already present in `gh-pages` (via #290). it'd be a good idea to start separating out your work on a single sample in unique named branches for clarity.